src/App.tsx

```diff
@@ -15,7 +16,7 @@ import i18n from './i18n';
 function App() {
   const Loader = () => (
     <div className="loading">
-      <div>loading...</div>
+      <CircularProgress />
     </div>
   );
```

Comment:
This should be defined outside App component, but I dont think it is necessary anymore, if anything the line 33 shuold be changed from Loader to CircularProgress